### PR TITLE
Create DAF.vue

### PR DESCRIPTION
**Pull request by weppsel**

### CODE REVIEW

File: src/plugins/builder/parts/DAF.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+import { Component } from 'vue-property-decorator';
+import PartBase from '../components/PartBase';
+@Component
+export default class Fridge extends PartBase {
```
Comment (steersbob) [inline src/plugins/builder/parts/DAF.vue:5]: Component name doesn't match file name.

File: src/plugins/builder/parts/DAF.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     <line x1="23" y1="4" x2="4" y2="4" stroke="black" />
+      <g>
+        <foreignObject
+          :transform="textTransformation([sizeX, sizeY], false)"
+          :width="squares(sizeX)"
+          :height="squares(sizeY)"
+        >
+          <div
+            class="text-white text-bold text-center text-h6 q-mt-xs"
+            style="max-width: 100%"line
```
Comment (steersbob) [inline src/plugins/builder/parts/DAF.vue:33]: What's the errant `line` doing here?

**Review by steersbob — COMMENTED** [review]
DAF.ts seems to be missing

**Comment by weppsel** [comment]
Ja klopt die moest ik nog maken. Moet ze in 1 pull request?

Verstuurd vanaf mijn iPhone

**Pull request closed without merging**